### DISCUSSION TITLE
Additional sensors (console temperature) in Home Assistant?

### DISCUSSION THREAD

**Discussion by dataworx22** [cytech/Home-Assistant-wundergroundpws#85] [General]
Hi, I'm using this integration in HA - thanks!!. I use a Davis VP2 PWS, uploading to Wunderground via Cumulus.  It works great, I was just wondering if I can push the console temperature to Home Assistant, to use in a couple of automations? 

I see there is no mention of internal (console) temperature on my station's page on Wunderground, but it appears that Cumulus does push that data. Is there perhaps a sensor name that I can add to my config HA to pull that data in?

**Comment by cytech** [comment]
not through the wunderground component
i also track my station locally with Weewx. Then created a mysql sensor to access the data thru Weewx

**Reply by dataworx22** [reply]
Thanks very much for taking the time to reply, I'll just get a dedicated temp/humidity sensor

**Comment by cytech** [comment]
closed as resolved

**Comment by cytech** [comment]
closed as resolved
